Resource_test.go

```diff
@@ -5,34 +5,36 @@ import (
 	"testing"
 )
 
-func TestFoldersTableSlice(t *testing.T) {
-	var foldersResBody *foldersResp
-	json.Unmarshal(fakeFolderByte, &foldersResBody)
-	slice1 := []string{"707", "The Foo Folder"}
-	slice2 := []string{"708", "The Baz Folder"}
-	expectedTable := [][]string{slice1, slice2}
-	matrixTestHelper(t, foldersResBody, expectedTable)
-}
-
-func TestSitesTableSlice(t *testing.T) {
-	var sitesResBody *sitesResp
-	json.Unmarshal(fakeSitesByte, &sitesResBody)
-	slice1 := []string{"1337", "Dyer"}
-	slice2 := []string{"42", "Situation"}
-	expectedTable := [][]string{slice1, slice2}
-	matrixTestHelper(t, sitesResBody, expectedTable)
-}
-
-func TestBrowsersTableSlice(t *testing.T) {
-	var browsersRespBody *browsersResp
-	err := json.Unmarshal(fakeClientsByte, &browsersRespBody)
-	if err != nil {
-		panic(err)
+func TestResourcesTableSlice(t *testing.T) {
+	var testCases = []struct {
+		resource      returnTable
+		json          []byte
+		expectedTable [][]string
+	}{
+		{
+			resource:      new(foldersResp),
```

Comment:
I don't think you need to actually include this in the table; instead you can put a temporary var within the `for` loop and before unmarshalling.